packages/app/.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       legacyDecorators: true
     }
   },
-  plugins: ['ember'],
+  plugins: ['ember', 'prettier', 'qunit'],
```

Comment:
intentional?

Reply:
nope, didnt do that on purpose I'll remove it